Apps/WebClient/src/ClientApp/src/views/validateEmail.vue

```diff
@@ -20,20 +21,29 @@ export default class ValidateEmailView extends Vue {
     @Action("checkRegistration", { namespace: "user" })
     checkRegistration!: (params: { hdid: string }) => Promise<boolean>;
 
-    private isLoading = false;
-    private isSuccess: boolean | null = null;
+    private isLoading = true;
+    private resultStatus: ResultType | null = null;
 
+    private get isSuccess() {
+        return this.resultStatus === ResultType.Success;
+    }
+    private get isActionRequired() {
```

Comment:
what does `isActionRequired` mean? (1) is the invite link expired or (2) invite link is still valid but user has not used it yet?

Reply:
same as above

Reply:
Looks like isActionRequired is just a wrapper for the internal checking of the result type for convenience. So the name should match that (as it does)